NEW_CONNECTION.md

```diff
@@ -0,0 +1,102 @@
+# Add New Data Source
+
+The following are the instructions for adding a new data source 
+
+## The following are the files that need to be updated
+
+1. app/constants/constants.js
+2. app/actions/sessions.js
+3. app/components/Settings/Tabs/Tab.react.js
+4. app/components/Settings/Settings.react.js
+5. backend/persistent/datastores/Datastores.js
+
+## The following are the new files that need to be added
+
+1. app/images
+2. backend/persistent/datastores/NameOfDatastoreToConnect.js
+
+# Detailed Implementation Instructions
+
```

Comment:
I'd remove this section and have all the subsections under `# Add New Data Source`.